File: jvm2/src/Model/World/GasVent.kt
package Model.World

import Model.Abstract.Resource
import Model.Location

class GasVent(location: Location) : Resource() {
    override var loc = location
    override var icon = "GV"
    override var health = 1000000
    override var minSal = 0
    override var gasSal = 1
    override var nrgSal = 0
}

File: dotnet/src/Model/World/GasVent.cs
using System;

namespace PlanetCS1 {
    class GasVent : Resource {
        public GasVent(Location location) {
            this.loc = location;
            this.icon = "GV";
            this.health = 1000000;
            this.minSal = 0;
            this.gasSal = 1;
            this.nrgSal = 0;
        }
    }
}